.drone.yml

```diff
@@ -6318,6 +6318,6 @@ kind: secret
 name: packages_secret_access_key
 ---
 kind: signature
-hmac: efffc2eb55bbfa1cebe950585df1f3f81d2f2a38311e2854f3ce08d1eec16fe3
+hmac: dcf24226fae30872050cdc031430374d811e6bbe13158ce0fbf234c90c1d83f9
```

Comment:
Why did that hmac change?

Reply:
This happens every time we make `.drone.yml` changes. Running `make drone`:
* Converts starlark files to `.drone.yml`.
* Lints `.drone.yml` file.
* Signs `.drone.yml` file. 
By changing the `hmac` signature, we can make sure that only people from Grafana Labs can make CI file changes. If that makes sense @zerok!